### PR TITLE
install after zat create

### PR DESCRIPTION
**Pull request by sandlerr**
### Description

When you `zat create` you almost always go and install the app afterwards. This does it automatically.

cc @zendesk/vegemite @fvzzy 

### Todo

Poll the job when the installation has requirements

### Risks

- [low] `zat create` failing

### CODE REVIEW

File: lib/zendesk_apps_tools/command.rb [hunk 1]
```diff
@@ -158,15 +148,20 @@ def server
...
 
       unless options[:zipfile]
-        app_name = JSON.parse(File.read(File.join options[:path], 'manifest.json'))['name']
+        app_name = manifest.name
       end
       app_name ||= get_value_from_stdin('Enter app name:')
       deploy_app(:post, '/api/v2/apps.json', name: app_name)
+      has_requirements = File.exist?(File.join(options[:path], 'requirements.json'))
+      say_status 'Install', 'installing'
+      install_app(has_requirements, app_id: cache.fetch('app_id'), settings: settings.merge(name: app_name))
```
Comment (danielbreves) [inline lib/zendesk_apps_tools/command.rb]: Shouldn't this be optional, especially considering it's new functionality being added to an existing command? Perhaps `zat create [-i | --install]`?

File: spec/lib/zendesk_apps_tools/command_spec.rb [hunk 2]
```diff
@@ -52,12 +52,18 @@
...
-        expect(File).to receive(:read) { '{ "name": "abc" }' }
+        expect(@command).to receive(:manifest).and_return(double('manifest', name: 'abc', original_parameters: [])).at_least(:once)
+        allow(@command).to receive(:options) { { clean: false, path: './', config: './settings.json' } }
 
         stub_request(:post, PREFIX + '/api/v2/apps.json')
           .with(body: JSON.generate(name: 'abc', upload_id: '123'),
                 headers: { 'Authorization' => 'Basic dXNlcm5hbWU6cGFzc3dvcmQ=' })
 
+        stub_request(:post, PREFIX + '/api/v2/apps/installations.json')
+          .with(body: JSON.generate(app_id: nil, settings: { name: 'abc' }),
```
Comment (fvzzy) [inline spec/lib/zendesk_apps_tools/command_spec.rb]: Maybe mock cache for app_id?

File: lib/zendesk_apps_tools/command.rb (hunk 1 above)
Comment (sandlerr) [inline lib/zendesk_apps_tools/command.rb]: going to bump a major and make it on by default

File: spec/lib/zendesk_apps_tools/command_spec.rb (hunk 2 above)
Comment (sandlerr) [inline spec/lib/zendesk_apps_tools/command_spec.rb]: done!

**Pull request merged**